Packages/com.unity.inputsystem/InputSystem/Devices/Keyboard.cs

```diff
@@ -155,17 +155,42 @@ public unsafe struct KeyboardState : IInputStateTypeInfo
         [InputControl(name = "OEM3", layout = "Key", bit = (int)Key.OEM3)]
         [InputControl(name = "OEM4", layout = "Key", bit = (int)Key.OEM4)]
         [InputControl(name = "OEM5", layout = "Key", bit = (int)Key.OEM5)]
-        [InputControl(name = "IMESelected", layout = "Button", bit = (int)Key.IMESelected, synthetic = true)]
+        [InputControl(name = "f13", displayName = "F13", layout = "Key", bit = (int)Key.F13)]
+        [InputControl(name = "f14", displayName = "F14", layout = "Key", bit = (int)Key.F14)]
+        [InputControl(name = "f15", displayName = "F15", layout = "Key", bit = (int)Key.F15)]
+        [InputControl(name = "f16", displayName = "F16", layout = "Key", bit = (int)Key.F16)]
+        [InputControl(name = "f17", displayName = "F17", layout = "Key", bit = (int)Key.F17)]
+        [InputControl(name = "f18", displayName = "F18", layout = "Key", bit = (int)Key.F18)]
+        [InputControl(name = "f19", displayName = "F19", layout = "Key", bit = (int)Key.F19)]
+        [InputControl(name = "f20", displayName = "F20", layout = "Key", bit = (int)Key.F20)]
+        [InputControl(name = "f21", displayName = "F21", layout = "Key", bit = (int)Key.F21)]
+        [InputControl(name = "f22", displayName = "F22", layout = "Key", bit = (int)Key.F22)]
+        [InputControl(name = "f23", displayName = "F23", layout = "Key", bit = (int)Key.F23)]
+        [InputControl(name = "f24", displayName = "F24", layout = "Key", bit = (int)Key.F24)]
+        [InputControl(name = "IMESelected", layout = "Button", bit = (int)KeyEx.RemapedIMESelected, synthetic = true)] // Use the last bit to hold IME selected state.
```

Comment:
Have you checked that this works for ImeSelected when using an older Unity version?

Reply:
yes, the remap is done by the package. The editor/player still write to the old location 111